src/code/fault.c

```diff
@@ -1,8 +1,8 @@
 #include <ultra64.h>
+#include <ultra64/controller.h>
```

Comment:
I don't know if this is planned to be changed at a later point with the header cleanup, but `ultra64/controller.h` should be included in `ultra64.h` imo, as well as `ultra64/pfs.h`.
Also not sure about the reason to not name them `os_cont.h` and `os_pfs.h`.

Reply:
I already planned to include all the missing libultra headers in `ultra64.h`. As for the header names I was actually wondering if weither or not we want to keep the `os_` prefix since it's been inconsistent so far.

Reply:
Right it's true that it's already inconsistent, and I'm not sure what we want to go with.
Either way if it's going to be changed soon then nvm this comment for this PR.